### PR TITLE
Update UG: Add Read features

### PR DESCRIPTION
**Pull request by ruo-x**
Features added are:
1. List all events
2. Read an individual event

### CODE REVIEW

**Review by brein62 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -181,6 +181,24 @@ Restrictions:
 Examples:
 * `ecreate n/food donation r/chef r/packer d/23-9-2023 1500 dsc/help food distribution m/50 potatoes b/50` creates an event with name `food donation`, roles needed `chef` and `packer`, event date `23rd September 2023, 3pm`, description `help food distribution`, materials needed `50 potatoes` and budget `$50`
 
+### List all events : `elist` [coming soon]
+Volunteer coordinators can see all the events they are organising. For each event, only the most important information will be shown: name, date and time, location.
+
+Format: `elist` (case-insensitive)
```
Comment (brein62) [inline docs/UserGuide.md:187]: don't think the case-insensitive part is required? if we want it that way, we can always add a note to the section under **Notes under the command format:** in the UG

File: docs/UserGuide.md [hunk 2]
```diff
@@ -181,6 +181,24 @@ Restrictions:
...
+
+### Read an individual event : `eshow` [coming soon]
+Volunteer coordinators can read up more about an individual event, to familiarize themselves with its requirements while planning for it.
+
+Format: `eshow EVENT_ID`
+
```
Comment (jordankanghm) [inline docs/UserGuide.md:193]: maybe can have a "Parameters" section for `EVENT_ID` to be consistent with the other commands.

**Review by jordankanghm — COMMENTED** [review]
Just some consistency suggestions!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -209,6 +209,24 @@ Restrictions:
 Examples:
 * `ecreate n/food donation r/chef r/packer d/23-9-2023 1500 dsc/help food distribution m/50 potatoes b/50` creates an event with name `food donation`, roles needed `chef` and `packer`, event date `23rd September 2023, 3pm`, description `help food distribution`, materials needed `50 potatoes` and budget `$50`
 
+### List all events : `elist` [coming soon]
```
Comment (jordankanghm) [inline docs/UserGuide.md:212]: Can change `List` to `Listing` to maintain consistency with other features.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -209,6 +209,24 @@ Restrictions:
...
+Volunteer coordinators can see all the events they are organising. For each event, only the most important information will be shown: name, date and time, location.
+
+Format: `elist`
+
+### Read an individual event : `eshow` [coming soon]
```
Comment (jordankanghm) [inline docs/UserGuide.md:217]: Same here, `Read` to `Reading`

**Review by jordankanghm — APPROVED** [review]
lgtm! just some minor nitpicking

File: docs/UserGuide.md (hunk 3 above)
Comment (brein62) [reply docs/UserGuide.md:212]: i agree

File: docs/UserGuide.md (hunk 4 above)
Comment (brein62) [reply docs/UserGuide.md:217]: i agree too

File: docs/UserGuide.md (hunk 3 above)
Comment (ruo-x) [reply docs/UserGuide.md:212]: oki

**Pull request merged**